pkg/config/config_template.yaml

```diff
@@ -873,6 +873,42 @@ api_key:
   #
   # log_file: /var/log/datadog/system-probe.log
 
+{{- if .SecurityModule }}
+
+##########################################
+## Security Agent Runtime Configuration ##
+##########################################
+
+# runtime_security_config:
+  ## @param enabled - boolean - optional - default: false
+  ## Set to true to enable the Security Runtime Module.
+  #
+  # enabled: false
+
+  ## @param socket - string - optional - default: /opt/datadog-agent/run/runtime-security.sock
+  ## The full path to the location of the unix socket where security runtime module is accessed.
+  #
+  # socket: /opt/datadog-agent/run/runtime-security.sock
+
+  ## @param policies - custom object - optional
+  ## Policy files
+  # policies:
+
+    ## @param dir - string - folder from where the policies will be loaded
+    ## Path from where the policy files will be loaded
+    #
+    dir: /etc/datadog-agent/runtime_security_agent.d
```

Comment:
Nit - maybe use dashes in paths - other paths seem to follow that convention?

Reply:
We followed the convention of folders in `cmd/agent/dist/conf.d`. Should we use the dash convention ?

Reply:
We already have `-` in `datadog-agent` in this path, and the socket path uses `-`.

Reply:
How about `/etc/datadog-agent/runtime-security.d`? Can we also set this as default in the config env binds?

Reply:
Changed to /etc/datadog-agent/runtime-security.d and updated default